core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -29,6 +29,7 @@ import org.apache.spark.executor.TaskMetrics
 import org.apache.spark.internal.Logging
 import org.apache.spark.scheduler._
 import org.apache.spark.status.api.v1
+import org.apache.spark.status.api.v1.{ApplicationAttemptInfo, ApplicationEnvironmentInfo, ApplicationInfo, RuntimeInfo}
```

Comment:
Just use `v1.blah` like existing code in this class.

Reply:
There is no ambiguity between ApplicationAttemptInfo and ApplicationAttempt classes in different versions anymore, so I thought the v1-s can be removed. I can undo that if they are still useful.

Reply:
It confusing to use different styles for different classes. Just using the existing style for all classes in that package.

Reply:
fixed